var/spack/repos/builtin/packages/py-pybrain/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPybrain(PythonPackage):
+    """PyBrain is the Swiss army knife for neural networking."""
+
+    homepage = "http://pybrain.org/"
+
+    url      = "https://github.com/pybrain/pybrain/archive/refs/tags/0.3.3.tar.gz"
+    git      = "https://github.com/pybrain/pybrain.git"
+
+    version('0.3.3.post', commit='dcdf32b')
```

Comment:
Is there any reason to check out from git commit instead of URL?

Reply:
https://github.com/pybrain/pybrain/issues/245